Sources/Controllers/MessagesViewController.swift

```diff
@@ -88,12 +88,14 @@ open class MessagesViewController: UIViewController {
     }
 
     open override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
```

Comment:
I don't think this is necessary, Apple docs says:
> Your view controller can override this method to make changes after the view lays out its subviews. The default implementation of this method does nothing.

Reply:
Emm, yep, default implementation does nothing, I just keep the form of call `super` when override method, I can move it.